scripts/run_tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+poetry run pytest tests
+poetry run pytest tests_e2e
```

Comment:
Would be slightly nicer to fix your tests so that you can just run `poetry run pytest`